### ISSUE TITLE
Radio Buttons Bug: Option Values Must Be Unique

### ISSUE THREAD

**Issue by omniacode** [CalderaWP/Caldera-Forms#893]
Have a simple setup with just two options, Male & Female. However, after clicking away from the field in the form builder, I am getting an error "Option values must be unique".

![screenshot](https://i.imgur.com/YD7Upk2.png)

**Comment by Shelob9** [comment]
I can't reproduce, but suspect it might be that another field is generating the error and its showing in the wrong place...

**Comment by qubez** [comment]
I have previously noted that the (hidden above) values are not always filled or copied from the display values. It may be that "show values" would reveal the blank entries in the example above.

**Comment by Shelob9** [comment]
@qubez That's the real bug, I've worked a bit with @omniacode on this and it looks like radio fields don't do that properly while other select fields do.

**Comment by qubez** [comment]
Same thing with dropdown lists, adding some more options to an existing list with "show values" unset gives you blank values and you are unable to save the form.

**Comment by Shelob9** [comment]
@qubez Yep, I saw same thing last night, not sure why but I'll try and figure it out. I updated the issue title to reflect that is the real issue here.

**Comment by bikechampion** [comment]
@Shelob9, can you confirm, that this bug will be solved soon? I started with Caldera today for the first time and ran directly in this problem.
